src/nvhttp.cpp

```diff
@@ -687,9 +687,19 @@ void applist(resp_https_t response, req_https_t request) {
   for(auto &proc : proc::proc.get_apps()) {
     pt::ptree app;
 
+    // Check if the "id" field is empty
+    if(!proc.id.empty()) {
+      // Use the "id" field if it is not empty
+      long long proc_id = std::stoll(proc.id);
+      app.put("ID", proc_id);
```

Comment:
It should be fine to just do `app.put("ID", proc.id)` here. Everything gets turned into a string anyway.